AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -35,7 +35,7 @@ To animate using Timeline Editor first can create a Timeline. You can then anima
 
 ![Timeline](images/timeline.png)
 
-Alternatively, you can drag and drop the scene object with the ABC asset onto an empty area within Timeline's Clip Editor this will create an Alembic Track with an Alembic Clip. If you are starting from an empty Timeline you will need to manual create the Alembic track first by right clicking the Timeline's Track view which is on the left hand side of the editor.
+Alternatively, you can drag and drop the scene object with the ABC asset onto an empty area within Timeline's Clip Editor this will create an Alembic Track with an Alembic Clip. If you are starting from an empty Timeline you may need to manually create the Alembic track first by right clicking the Timeline's Track view on the left hand side of the editor.
 
 ![Timeline](images/timeline2.png)
 
```

Comment:
Another occurrence of ABC at line 44 (instead of Alembic). Let's decide on whether we want to change all occurrences of  ABC to Alembic and please search and replace them all